infrastructure/simulation-tool/test/operations/changePubKey.test.ts

```diff
@@ -0,0 +1,219 @@
+import { Wallet as RollupWallet, Transaction } from '@rsksmart/rif-rollup-js-sdk';
+import sinon from 'sinon';
+import sinonChai from 'sinon-chai';
+import {
+    PreparedPubKeyChange,
+    executePubKeyChange,
+    executePubKeyChanges,
+    generatePubKeyChanges,
+    preparePubKeyChange,
+    resolveTransaction,
+    resolvePubKeyChanges,
+    ResolvedPubKeyChange
+} from '../../src/operations/changePubKey';
+import { expect, use } from 'chai';
+import { constants } from 'ethers';
+import { RollupWalletGenerator } from '../../src/utils/wallet.utils';
+
+use(sinonChai);
+
+describe('changePubKey', () => {
+    afterEach(() => {
+        sinon.restore();
+    });
+
+    describe('preparePubKeyChange', () => {
+        let senderWallet: RollupWallet;
+        beforeEach(() => {
+            senderWallet = sinon.createStubInstance(RollupWallet);
+        });
+        it('should return object with "from" set to the sender parameter', () => {
+            const expectedSender = senderWallet;
+            const { from } = preparePubKeyChange(expectedSender);
+
+            expect(from).to.deep.equal(expectedSender);
+        });
+
+        it('should return object with "ethAuthType" set to "ECDSA"', () => {
+            const expectedValue = 'ECDSA';
+            const { ethAuthType } = preparePubKeyChange(senderWallet);
+
+            expect(ethAuthType).to.deep.equal(expectedValue);
+        });
+
+        it('should return object with "feeToken" set to zero address', () => {
+            const expectedFeeToken = constants.AddressZero;
+            const { feeToken } = preparePubKeyChange(senderWallet);
+
+            expect(feeToken).to.deep.equal(expectedFeeToken);
```

Comment:
I know it doesn't hurt, but we don't probably need deep here

Reply:
done